docs/general/commands.md

```diff
@@ -171,7 +171,7 @@ about the bundle size.
 npm run lint
 ```
 
-Lints your JavaScript and CSS.
+Lints your JavaScript.
 
 ### JavaScript
```

Comment:
Looks kind of weird/redundant. How about leaving both CSS and JS sections in the doc for future use, but commenting them out `<!-- ... --> ` until style linting get worked out?

![image](https://cloud.githubusercontent.com/assets/1481063/21744527/30984644-d4cc-11e6-8d33-4ce381ddbba0.png)